about/get-involved.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
@megahirt Could you review the content of `get-involved.php` as a perspective from someone outside the team?